frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -63,6 +64,15 @@ public String getActionName() {
   public void update(AnActionEvent anActionEvent) {
     super.update(anActionEvent);
     syncToRemoteStatusDependentActionUpdate(anActionEvent);
+
```

Comment:
While you're here, I'd change `getActionName` so that it returns `"Reset to remote"`; this will make the captions that use this action name more clear